bodhi/validators.py

```diff
@@ -489,6 +489,16 @@ def validate_releases(request):
         request.validated["releases"] = validated_releases
 
 
+def validate_bugs(request):
+    bugs = request.validated.get('bugs')
+    if bugs:
+        try:
+            request.validated['bugs'] = map(int, bugs)
+        except ValueError:
```

Comment:
I'm surprised this is a ``ValueError``... I'd expect it to be a ``TypeError``.  I just checked in `bpython` and it's true!